src/Pmi/Controller/BiobankController.php

```diff
@@ -33,15 +33,24 @@ public function homeAction(Application $app)
 
     public function participantsAction(Application $app, Request $request)
     {
+        $bioBankIdPrefix = $app->getConfig('biobank_id_prefix');
+        $constraints = [
+            new Constraints\NotBlank(),
+            new Constraints\Type('string')
+        ];
+        if (!empty($bioBankIdPrefix)) {
+            $bioBankIdPrefix = preg_quote($bioBankIdPrefix, '/');
+            $constraints[] = new Constraints\Regex([
+                'pattern' => "/^{$bioBankIdPrefix}\d+$/",
+                'message' => "Invalid biobank ID. Must be in the format of {$bioBankIdPrefix}000000000"
```

Comment:
Sorry, one more detail - the prefix in the validation message should not be preg_quoted.